app/assets/javascripts/initializers/initializeEllipsisMenu.js

```diff
@@ -47,7 +47,9 @@ function onXhrSuccess(form, article, values) {
     toggleNotifications(submit, submitValue);
   }
 
-  article.querySelector('.js-ellipsis-menu').classList.add('hidden');
+  article
+    .getElementsByClassName('.js-ellipsis-menu')[0]
```

Comment:
@rhymes should this be:

```suggestion
    .getElementsByClassName('js-ellipsis-menu')[0]
```